orc8r/cloud/deploy/bare-metal-ansible/ansible_vars.yaml

```diff
@@ -0,0 +1,66 @@
+ansible_user: ubuntu
+
+helm_enabled: true
+kube_network_plugin: calico
+kube_network_plugin_multus: true
+
+kube_apiserver_port: 6443
+kube_pods_subnet: 10.233.64.0/18
+kube_network_node_prefix: 24
+kube_service_addresses: 10.233.0.0/18
+kubeadm_control_plane: "true"
+etcd_kubeadm_enabled: true
+
+enable_nodelocaldns: true
+resolvconf_mode: host_resolvconf
+
+# Copies kubectl and kubeconfig to your ansible host
+kubectl_localhost: true
+kubeconfig_localhost: true
+
+# Change these to your private nameservers if needed
+nameservers:
+  - 8.8.8.8
+  - 8.8.4.4
+upstream_dns_servers:
+  - 8.8.8.8
+  - 8.8.4.4
+
+# If we have a docker registry without trusted SSL or http only, add here
+# docker_insecure_registries:
+#   - private_registry:5000
+
+## Kubernetes LoadBalancer
+## Enabling a loadbalancer is optional, but provides HA for external access to Kubernetes API
+## loadbalancer_apiserver and vrrp_nic need to be set if you want to enable this feature.
+
+# Change to match the network of nodes where you want to run loadbalancer
```

Comment:
instead of commenting this out, can you gate this by using a variable `enable_loadbalancer` instead ?